src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -1272,22 +1273,23 @@ private CompressedPagesResult PrepareToWriteToJournal(LowLevelTransaction tx)
                 pagesCountIncludingAllOverflowPages += page.NumberOfPages;
             }
 
-            // We want to include the Transaction Header straight into the compression buffer.
+            bool performCompression = pagesCountIncludingAllOverflowPages > 200; // TODO : Make this variable and configurable
```

Comment:
This translate to about 1.5 MB. Might be interesting to set it to 512 / 256 KB, that way we'll say on small tx, I'm not sure how common > 1.5 MB per tx will be.